Maple2.Model/Game/Item/ItemBlueprint.cs

```diff
@@ -1,34 +1,45 @@
-﻿using Maple2.PacketLib.Tools;
+﻿using Maple2.Model.Enum;
+using Maple2.PacketLib.Tools;
 using Maple2.Tools;
 
 namespace Maple2.Model.Game;
 
 public sealed class ItemBlueprint : IByteSerializable, IByteDeserializable {
+    public long BlueprintUid;
+    public int Length;
+    public int Width;
+    public int Height;
+    public DateTimeOffset CreationTime;
+    public BlueprintType Type = BlueprintType.Original;
+    public long AccountId;
+    public long CharacterId;
+    public string CharacterName = "";
```

Comment:
**suggestion:** Reconsider using empty strings for nullable properties

Some properties in ItemBlueprint, like CharacterName, are initialized with empty strings instead of being nullable. Consider making these properties nullable (string?) if they can legitimately be null, as this more accurately represents the data and aligns with best practices for nullable reference types.